### PR TITLE
feature/move_search

### PR DESCRIPTION
**Pull request by jonhindle**
### Context

> Does this issue have a Trello card?
https://trello.com/c/bWDNgSgo/772-move-the-search-bar-into-the-header

> If this is an issue, do we have steps to reproduce?
n/a

### Intent

> What changes are introduced by this PR that correspond to the above card?
search bar relocated to top bar
page navigation recoloured and resized
section break restyled

> Would this PR benefit from screenshots?
Screenshot 2022-03-22 at 23 10 49

### Considerations

> Is there any additional information that would help when reviewing this PR?
Screenshot 2022-03-22 at 23 10 13
n/a

> Are there any steps required when merging/deploying this PR?
n/a

### Checklist

- [ ] This PR contains **only** changes related to the above card
- [ ] Tests have been added/updated to cover the change
- [ ] Documentation has been updated where appropriate
- [ ] Tested in Development

### CODE REVIEW

File: assets/sass/_general.scss [hunk 1]
```diff
@@ -128,8 +128,26 @@
...
+.govuk-hub-navigationUserDetailsBar {
+  display: flex;
+  flex-wrap: nowrap;
+  justify-content: space-between;
+  align-items: center;
+  margin: 0;
+}
+
+.govuk-hub-userDetails {
+  padding: 10px 0;
```
Comment (andrewrlee) [inline assets/sass/_general.scss:147]: meh.. any govuk spacing constant we could use here?

**Review by andrewrlee — DISMISSED** [review]
LGTM codewise

**Comment by andrewrlee** [comment]
Just checked a few pages 

404 page is not looking too healthy - bit squashed up title and missing categories

Screenshot 2022-03-31 at 23 42 03
 

Search results is bit squashed up too
Screenshot 2022-03-31 at 23 44 31

**Comment by andrewrlee** [comment]
are we in a position we can get rid of pop tops from establishment.json etc..?

**Review by andrewrlee — APPROVED** [review]
LGTM

**Review by davidthomas22 — COMMENTED** [review]
LGTM

**Pull request merged**